lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -21,6 +21,41 @@ def self.email(params = {}, _secrets = {}, _context = {})
 
         private_class_method def self.email_cleanup(*)
         end
+
+        def self.provision_execute(_params = {}, _secrets = {}, context = {})
+          return unless context.execution["_object_type"] == "ServiceTemplateProvisionTask"
+
+          ::MiqRequestTask.find(context.execution["_object_id"])&.execute
+
+          {"miq_request_task_id" => request_task_id}
```

Comment:
```
#<NameError: undefined local variable or method `request_task_id' for ManageIQ::Providers::Workflows::BuiltinMethods:Class

          {"miq_request_task_id" => request_task_id}
```

Reply:
Think this is available in line 28 now? (old comment?)

Reply:
This comment was from before the most recent push which fixed it